common/source/docs/common-optional-hardware.rst

```diff
@@ -22,7 +22,8 @@ information related to Autopilot selection see :ref:`Autopilot Hardware Options
 [site wiki="plane"]
     Airspeed Sensor <airspeed>
 [/site]
-    Battery Monitors (aka Power Modules) <common-powermodule-landingpage>
+    Power Modules <common-powermodule-landingpage>
```

Comment:
this list is alphabetical....if you wish to split into two groups, list them in order further down